dist/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM alpine:3.20 AS builder
+RUN apk update && apk add --no-cache \
+    build-base \
+    curl \
+    git \
+    linux-headers && \
+    git clone -b \
```

Comment:
Better split `git clone` and `apk add`, see https://github.com/tazihad/byedpi-docker/blob/main/Dockerfile

Reply:
OK